pkg/provider/utils.go

```diff
@@ -3,7 +3,10 @@ package provider
 import (
 	"errors"
 
+	"fmt"
+
 	"github.com/EnterpriseDB/terraform-provider-biganimal/pkg/api"
+	diag2 "github.com/hashicorp/terraform-plugin-framework/diag"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```

Comment:
```suggestion
	sdkdiag "github.com/hashicorp/terraform-plugin-sdk/v2/diag"
```
As we're introducing frameworkdiag, let's name this one as sdkdiag, and update the fromBigAnimalErr function as well. Thank you 🙏